assets/chezmoi.io/docs/reference/templates/variables.md

```diff
@@ -32,6 +33,15 @@ chezmoi provides the following automatically-populated variables:
 | `.chezmoi.windowsVersion`     | object   | Windows version information, if running on Windows                                                                                                    |
 | `.chezmoi.workingTree`        | string   | The working tree of the source directory                                                                                                              |
 
+`.chezmoi.darwinVersion` contains the following keys populated from the
+`/System/Library/CoreServices/SystemVersion.plist`.
+
+| Key                         | Type    |
+| --------------------------- | ------- |
+| `buildVersion`              | string  |
+| `productName`               | string  |
+| `productVersion`            | integer |
```

Comment:
This should not be an `integer` but a `string`. If you want to parse it into `productMajorVersion` and `productMinorVersion` integer values, that is acceptable.

Reply:
This is a typo from copy pasting. Should be a string.

Reply:
It might be worth parsing it into `productMajorVersion` and `productMinorVersion` as well as noted. That will make version comparison easier.